File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    while (1) {
        int input = get_int("Height: ");
        if (input < 1 || input > 8) {
            break;
        }
        for (int i = 1; i <= input; i++) {
            for (int j = 0;j < input - i; j++) {
                printf(" ");
            }
            for (int j = 0; j < i; j++) {
                printf("#");
            }
            printf("  ");
            for (int j = 0; j < i; j++) {
                printf("#");
            }
            printf("\n");
        }
        break;
    }
    return 0;

}

File: cs50/sentimental-mario-more/mario.py
# TODO
while True:
    try:
        height = int(input("Height: "))
        if 1 <= height <= 8:
            for i in range(1, height + 1):
                for j in range(height - i):
                    print(" ", end="")
                for j in range(i):
                    print("#", end="")
                print("  ", end="")
                for j in range(i):
                    print("#", end="")
                print("")
            break
    except:
        continue
